cli/azd/pkg/output/ux/uxitem.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package ux
+
+import "github.com/azure/azure-dev/cli/azd/pkg/output"
+
+type UXItem interface {
```

Comment:
I'm not sure what's conventional, but I'd prefer the casing `UxItem` over `UXItem`, which makes it hard to distinguish abbreviation from word separation.